app-nodejs-html5/app.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const app = express();
+const port = 8080;
+
+app.use('/app-html5', express.static('src/main/webapp/'));
+
+app.listen(port, () => {
+  console.log('app-nodejs-html5 listening on port 8080');
```

Comment:
Instead of hard-coding the port number in this string you can interpolate the existing constant.
```suggestion
  console.log(`app-nodejs-html5 listening on port ${port}`);
```